### DISCUSSION TITLE
tutorials/getting-started-with-graphos/07-using-defer

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#56] [course feedback]
# tutorials/getting-started-with-graphos/07-using-defer

https://www.apollographql.com/tutorials/getting-started-with-graphos/07-using-defer

**Comment by forresty** [comment]
I am seeing `unknown directive @defer` error; did I miss any step? 

```json
{
  "errors": [
    {
      "message": "Unknown directive \"@defer\".",
      "locations": [
        {
          "line": 14,
          "column": 7
        }
      ],
      "extensions": {
        "code": "GRAPHQL_VALIDATION_FAILED"
      }
    }
  ]
}
```

**Reply by forresty** [reply]
Ah, never mind, I was querying against the subgraph instead of the supergraph. It works now. Sorry!

**Reply by lizhennessy** [reply]
Glad to hear you got it solved @forresty ! Please don't hesitate to reach out if you have any other questions.
